### PR TITLE
Pia 3089 update colors no results

### PR DESCRIPTION
**Pull request by chriskryn**
- updated colors of no result screen
- fit help screens to bottom border

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift [hunk 1]
```diff
@@ -1143,6 +1143,7 @@ public final class GiniBankConfiguration: NSObject {
     
     @objc public var primaryButtonShadowRadius: CGFloat = 14
     
+
```
Comment (vizaknaidavid) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift:1146]: This extra line is not needed

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpFormatsViewController.swift [hunk 2]
```diff
@@ -65,7 +70,7 @@ final class HelpFormatsViewController: UIViewController {
         tableView.delegate = self.dataSource
         tableView.dataSource = self.dataSource
         tableView.estimatedRowHeight = tableRowHeight
-        tableView.showsVerticalScrollIndicator = false
+        tableView.showsVerticalScrollIndicator = true
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpFormatsViewController.swift:73]: The default value is true, this line can be deleted entirely

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpImportViewController.swift [hunk 3]
```diff
@@ -58,6 +58,7 @@ class HelpImportViewController: UIViewController {
         tableView.allowsSelection = false
         tableView.backgroundColor = UIColor.clear
         tableView.tableFooterView = UIView()
+        tableView.showsVerticalScrollIndicator = true
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpImportViewController.swift:61]: Same

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpTipsViewController.swift [hunk 4]
```diff
@@ -63,7 +68,7 @@ public final class HelpTipsViewController: UIViewController {
                 nibName: "HelpTipCell",
                 bundle: giniCaptureBundle()),
             forCellReuseIdentifier: HelpTipCell.reuseIdentifier)
-        tableView.showsVerticalScrollIndicator = false
+        tableView.showsVerticalScrollIndicator = true
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpTipsViewController.swift:71]: Same

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.swift [hunk 5]
```diff
@@ -57,7 +57,6 @@ extension UIColor {
         static let helpBackground = UIColorPreferred(named: "helpBackground")
         static let separator = UIColorPreferred(named: "separator")
         static let subheadline = UIColorPreferred(named: "subheadline")
-        static let systemBlue = UIColorPreferred(named: "systemBlue")
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.swift:60]: If you remove a color from here, make sure to remove the color asset as well

**Review by vizaknaidavid — CHANGES_REQUESTED** [review]

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift [hunk 6]
```diff
@@ -1153,19 +1154,20 @@ public final class GiniBankConfiguration: NSObject {
...
     
-    @objc public var primaryButtonBackgroundColor: UIColor = UIColor.GiniBank.systemBlue ?? UIColor.clear
+    @objc public var primaryButtonBackgroundColor: UIColor = GiniColor(light: UIColor.GiniBank.accent1, dark: UIColor.GiniBank.accent1).uiColor()
     
     /**
      Indicates title color for primary buttons within the screens
      */
     
-    @objc public var primaryButtonTitleColor: UIColor = UIColor.GiniBank.labelWhite ?? UIColor.clear
+    @objc public var primaryButtonTitleColor: UIColor = GiniColor(light: UIColor.GiniBank.light1, dark: UIColor.GiniBank.light1).uiColor()
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift:1169]: @chriskryn what do you think shall we do it GiniColor?

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Please apply GiniColor for buttons configuration and use uiColor() inside the creation code of this button 😉

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.swift (hunk 5 above)
Comment (zladzeyka) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.swift:60]: I was thinking we will do the cleanup for all colors? @chriskryn could you please put it back 😉

**Review by zladzeyka — APPROVED** [review]
👍🏻

**Pull request merged**